NightscoutServiceKit/RemoteCommands/Extensions/BolusRemoteNotification.swift

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
We need to translate between our Nightscout codable DTOs to RemoteCommand types that Loop can use. These extensions will provide the translation.